django_usaid_clinicfinder/clinicfinder/serializers.py

```diff
@@ -22,7 +23,7 @@ class PointOfInterestSerializer(HyperlinkedModelSerializer):
 
     class Meta:
         model = PointOfInterest
-        fields = ('url', 'id', 'data', 'location', 'created_at', 'updated_at')
+        fields = ('url', 'id', 'data', 'location')
```

Comment:
That serializer was never used in a relationship (FK) that was posted. I'm going to take them out incase they are